### DISCUSSION TITLE
Shuttle lights A has dirty textures with some floating pixels

### DISCUSSION THREAD

**Discussion by Holneer** [FakeFishGames/Barotrauma#14986] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![image](https://github.com/user-attachments/assets/5fec5a61-1a84-4c68-9246-375b13981eff)

### Reproduction steps

1. Have eyes.
2. Look at Shuttle Lights A.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.18.1 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This has been addressed in our private repo now, fix coming up in a future update!
